.github/pull_request_template.md

```diff
@@ -34,7 +34,7 @@
 * [ ] Changes are limited to a single goal - _eschew scope creep!_
 * [ ] _All_ future TODOs are captured in issues, which are referenced
       in code comments.
-* [ ] An assignee and type of change labels have been added.
+* [ ] An assignee and all relevant type-of-change labels have been added.
```

Comment:
I don't think external contributors can assign PRs to anyone.  Should we change this to:
```suggestion
* [ ] All relevant type-of-change labels have been added.
```